index.html

```diff
@@ -1,11 +1,244 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
```

Comment:
**[Optional]** 🍀 Friendly note, in your project you used 4 spaces in your document, in future projects you will be required to use a 2 space indentation to follow html conventions